nntrainer/tensor/tensor.h

```diff
@@ -203,6 +197,8 @@ class Tensor {
    * @note      This constructor copies vector again. needs refactoring
    * @param[in] d data for the Tensor with batch size one
    * @param[in] t_type Tensor Type
+   * @todo      It is more desirable to move this implementaton into
+   *            `tensor.cpp`, for it requires half_tensor.h
```

Comment:
I see all other data type implementations have been moved to .cpp except for half tensor. could you explain why you left it as todo?

Reply:
I attempted to relocate it to 'tensor.cpp' with `#ifdef`, but encountered a linking error during compilation. Thus, I decided to leave it unchanged in the current version and will investigate soon.